src/dive.ts

```diff
@@ -159,15 +159,15 @@ export default class DIVE {
 
         // apply perspective camera settings
         if (settingsDelta.perspectiveCamera) {
-            this.perspectiveCamera.fov = settingsDelta.perspectiveCamera.fov;
-            this.perspectiveCamera.near = settingsDelta.perspectiveCamera.near;
-            this.perspectiveCamera.far = settingsDelta.perspectiveCamera.far;
+            if (settingsDelta.perspectiveCamera.fov !== undefined) this.perspectiveCamera.fov = settingsDelta.perspectiveCamera.fov;
+            if (settingsDelta.perspectiveCamera.near !== undefined) this.perspectiveCamera.near = settingsDelta.perspectiveCamera.near;
+            if (settingsDelta.perspectiveCamera.far !== undefined) this.perspectiveCamera.far = settingsDelta.perspectiveCamera.far;
```

Comment:
should default settings values be assigned in case of undefined? just comment in case it does, approving the PR anways 👍

Reply:
The idea is to use a Partial<"Settings">. So instead of providing full information about all possible settings, you can just define one particular settings that you want to set. The other ones are `undefined` then. So we have to check if the settings are `undefined` and only of they are defined, they have to be set.

Does that make sense to you?